privacyidea/static/components/config/controllers/configControllers.js

```diff
@@ -34,19 +36,52 @@ myApp.controller("policyListController", function($scope, $stateParams,
         });
     };
 
+    $scope.delPolicyAsk = function(policyName) {
+        $scope.confirm(
+            $scope.confirm_action_levels["severe"],
+            "Delete Policy",
```

Comment:
Does this mean the `[translate]`-Attributes should be removed for the dynamic fields in the template? Should I add the translation calls to `$scope.confirm()`, to avoid code duplication, or do you this that would reduce clarity?

Reply:
Fixed.